.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+## Feature Description
+
+<!-- Please describe clear and concisely which problem the feature would solve or which publisher needs it would address. -->
+
+---------------
+
+_Do not alter or remove anything below. The following sections will be managed by moderators only._
+
+## Acceptance criteria
+
+* <!-- One or more bullet points for acceptance criteria. -->
+
+## Implementation Brief
+
+* <!-- One or more bullet points for how to technically resolve the issue. -->
```

Comment:
Should mentioned that attachments are ok? What if the implementation brief is robust, and is better suited for its own document.